### ISSUE TITLE
Faceted Browse results are not present in accessibility tree

### ISSUE THREAD

**Issue by erinchambers** [CDRH/theme_nebraskaland#5]
As a result of an `aria-hidden="true"` in the module's base code, the faceted browse results and pagination are absent from the accessibility tree. Screen readers cannot access the search results; they go straight from the facets to the footer.

Added to issue here: https://github.com/omeka-s-modules/FacetedBrowse/issues/68

Posted in Omeka S forums and Sharon Leon replied that they are working on a fix.

**Comment by erinchambers** [comment]
Check in with Omeka S team - looks like progress has been made, but not sure if they're ready for release just yet. If it comes out after we hand the site off to Nebraska Game & Parks, we'll want to keep an eye out and then touch base with Jackie when ready. We will probably need to help upgrade the module to make sure our template overrides don't cause conflicts.

**Comment by erinchambers** [comment]
This was fixed in the latest Omeka S release (hooray!)
